vendor/bat-native-ads/src/bat/ads/internal/ml/transformation/normalization_transformation_unittest.cc

```diff
@@ -10,7 +10,6 @@
 
```

Comment:
Surprisingly, this file already contains tests for both sparse/dense(trivial) cases. So looks like no extra tests are needed.
But I've added a few for `Normalize()`.